scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_2.txt

```diff
@@ -0,0 +1,34 @@
+Modules from the same toolchain always work together, and modules from a
+\*different version of the same toolchain\* never work together.
+The above set of modules works together: an overview of compatible toolchains can be found here:
+https://docs.easybuild.io/en/latest/Common-toolchains.html#overview-of-common-toolchains.
```

Comment:
that url is way too detailed. also the rows should be resorted to have most recent modules first (or cleanup the really old ones)

Reply:
the issue is probaly that the subscetion reference is broken, and currentl link points to the begin of page.
`https://docs.easybuild.io/common-toolchains/#common_toolchains_overview` is probably what you want